src/main/java/de/zalando/ep/zalenium/container/DockerContainerClient.java

```diff
@@ -272,12 +265,14 @@ public ContainerCreationStatus createContainer(String zaleniumContainerName, Str
         extraHosts.addAll(hubExtraHosts);
 
         HostConfig.Builder hostConfigBuilder = HostConfig.builder()
-            .appendBinds(binds)
-            .networkMode(networkMode)
-            .extraHosts(extraHosts)
-            .autoRemove(true)
-            .storageOpt(storageOpt)
-            .privileged(isZaleniumPrivileged);
+                .appendBinds(binds)
+                .networkMode(networkMode)
+                .extraHosts(extraHosts)
+                .autoRemove(true)
+                .storageOpt(storageOpt)
+                .nanoCpus(Long.valueOf(cpuLimit))
```

Comment:
Please only add this if the environment variables are present, we should not have defaults since every user is different. Just nest it into an "if".